app/assets/javascripts/discourse/app/components/scrolling-post-stream.js

```diff
@@ -319,7 +319,7 @@ export default class ScrollingPostStream extends MountWidget {
     };
     document.addEventListener("touchmove", this._debouncedScroll, opts);
     window.addEventListener("scroll", this._debouncedScroll, opts);
-    this._scrollTriggered();
+    next(() => this._scrollTriggered());
```

Comment:
This worries me a little, since it applies to old post stream, and this code is super sensitive.

Happy to try it, but if it causes any issues we may need to tie it to the feature flag.